canonical-web-and-design/canonical.com
# Hide overview tab on /careers/all and add 404 & 500 pages

## Done

- Hide `overview` tab on `/careers/all`
- Add 404 & 500 pages
- Remove secondary nav on all `/careers` pages
- Add drop down for mobile navigation
- Update image on `/careers/travel`
- Make navigation sticky on home page
- Remove navigation drop downs on home page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers/all
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure the overview tab is not visible
- Go to http://0.0.0.0:8002/404 and http://0.0.0.0:8002/500 and see a sensible content on the page


## Issue / Card

Fixes #108
